src/Lykke.Service.BlockchainApi.Sdk/Domain/DepositWallets.cs

```diff
@@ -0,0 +1,201 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AzureStorage;
+using AzureStorage.Tables;
+using Lykke.AzureStorage.Tables;
+using Lykke.AzureStorage.Tables.Entity.Annotation;
+using Lykke.AzureStorage.Tables.Entity.ValueTypesMerging;
+using Lykke.Common.Log;
+using Lykke.SettingsReader;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Lykke.Service.BlockchainApi.Sdk.Domain
+{
+    public class DepositWalletEntity : TableEntity
+    {
+        public static string Partition(string address) => address;
+        public static string Row() => "";
+
+        public DepositWalletEntity() {}
+        public DepositWalletEntity(string address) => (PartitionKey, RowKey) = (Partition(address), Row());
+
+        [IgnoreProperty]
+        public string Address { get => PartitionKey; }
+    }
+
+    [ValueTypeMergingStrategyAttribute(ValueTypeMergingStrategy.UpdateAlways)]
+    public class DepositWalletBalanceEntity : AzureTableEntity
+    {
+        public static string Partition(string address) => address;
+        public static string Row(string assetId) => assetId;
+
+        public DepositWalletBalanceEntity() {}
+        public DepositWalletBalanceEntity(string address, string assetId) => (PartitionKey, RowKey) = (Partition(address), Row(assetId));
+
+        [IgnoreProperty] 
+        public string  Address     { get => PartitionKey; }
+        [IgnoreProperty] 
+        public string  AssetId     { get => RowKey; }
+        public decimal Amount      { get; set; }
+        public long    BlockNumber { get; set; }
+    }
+
+    [ValueTypeMergingStrategyAttribute(ValueTypeMergingStrategy.UpdateAlways)]
```

Comment:
Almost the same thing as above, but you always specify all properties on InsertOrMerge below so you don't need Merge, you actually alwasy Replace existing entity.

Reply:
Same as above